package.json

```diff
@@ -56,6 +56,7 @@
     "test.watch-all": "jest --watchAll --coverage"
   },
   "devDependencies": {
+    "@aivenio/tsc-output-parser": "^2.1.1",
```

Comment:
I added this as a dev dependency -- however, we could simply install this package manually in the action if we want to avoid increasing the number of dependencies

Reply:
that makes sense, I'll go ahead and make that change